UPGRADE.md

```diff
@@ -307,6 +307,29 @@ in your `app/config.php`, it is no longer registered by default.
 ],
 ```
 
+### Update lighthouse.guard configuration
+
+The `lighthouse.guard` configuration key was renamed to `lighthouse.guards` and expects an array.
+
+```diff
+- 'guard' => 'api',
++ 'guards' => ['api'],
+```
+
+If `lighthouse.guards` configuration is missing,
+the default Laravel authentication guard will be used (`auth.defaults.guard`).
+
+### Update `@auth` and `@whereAuth` directives
+
+The `guard` attribute, on `@auth` and `@whereAuth` directives, has been renamed `guards` and expects an array.
```

Comment:
```suggestion
The `guard` argument on `@auth` and `@whereAuth` directives has been renamed to `guards` and now expects a list instead of a single string.
```